packages/vue/docs/getting-started.md

```diff
@@ -10,7 +10,7 @@ In order to use it in custom projects you need the following webpack loaders: `c
 Storefront UI is installed as a dependency to your project:
 
 ```bash
-npm install --save @storefront-ui/vue
+yarn install --save @storefront-ui/vue
```

Comment:
AFAIC, it is _not_ necessary to use `yarn` when _using_ SFUI; you can also use it in your npm projects. But it is necessary for contributors. Am I right on this, @pspaczek / @patzick ?

Reply:
I agree with you on this. The best thing to do would be to have both ways there so the user can chose which one they want.